UIkitDZ/New Group/StopWatchViewController.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  ViewController.swift
+//  UIkitDZ
+//
+//  Created by Timofey Privalov on 30.06.2021.
+//
+
+import UIKit
+
+final class StopWatchViewController: UIViewController {
+    private var timer = Timer()
+    private var (hour, minute, second, fraction) = (0, 0, 0, 0)
+
+    @IBOutlet weak var ButtonReset: UIButton!
+    @IBOutlet weak var startButton: UIButton!
+    @IBOutlet weak var timerMainLabel: UILabel!
+    @IBOutlet weak var fractionLabel: UILabel!
+    @IBOutlet var linesView: [UIView]!
+    @IBOutlet var lapsLabel: [UILabel]!
+
+    //MARK: logic for start | stop button
+    @IBAction private func startButton(_ sender: UIButton) {
+
+        if let button = sender as? UIButton {
+            if button.titleLabel?.text == "Start" {
+                timerGo()
+                button.setTitle("Pause", for: .normal)
+                button.setImage(UIImage(named: "stop.png"), for: .normal)
+            } else {
+                timer.invalidate()
+                for (i,k) in zip(lapsLabel, linesView) {
+                    if i.text == "00:00:00,00" {
+                        i.text = "\(hour):\(minute):\(second),\(fraction)"
+                        i.textColor = .white
+                        k.backgroundColor = .darkGray
+                        break
+                    }
+                }
+                button.setTitle("Start", for: .normal)
+                button.setImage(UIImage(named: "start.png"), for: .normal)
+            }
+        }
+    }
+    //MARK: logic for button reset
+    @IBAction private func resetTimer(_ sender: UIButton) {
+        self.timer.invalidate()
+        (hour, minute, second, fraction) = (0, 0, 0, 0)
+        timerMainLabel.text = "00:00:00,"
+        fractionLabel.text = "00"
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+    //MARK: createing timer
+    private func timerGo() {
+        timer = Timer.scheduledTimer(timeInterval: 0.01, target: self, selector: #selector(keepTimer), userInfo: nil, repeats: true)
+    }
+    //MARK: -creating logic of timer for labels
```

Comment:
```suggestion
    //MARK: -creating logic of timer for labels 
```
в 3 й раз пишу такие марки не пишем прием!